spotify-artist-viewer/Models/SAArtist.m

```diff
@@ -12,15 +12,18 @@ @implementation SAArtist
 @synthesize artistUri;
 @synthesize artistName;
 @synthesize imgURL;
+@synthesize bio;
```

Comment:
You do not have to @synthesize your properties. The getter/setter is created atomically just by using the @property to define the property
.